docs/src/main/tut/enum.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title:  "Enum"
+source: "core/src/main/scala/Enum.scala"
+---
+# Enum
+
+`Enum` represents discrete operations that can be performed on a type `A`
+
+These operations are presented by the following functions.
+
+- `def succ(x)`:		Returns the successor value of `x`
+- `def pred(x)`: 	Returns the predecessor value of `x`
+- `def adj(i, j)`:	Returns if `i` and `j` are consecutives ( `succ(i) is j`)
+
+An example of the discrete operation on interger values could be: 
+
+```
```

Comment:
please make this "```tut" not just "```" so that it is compiled, this is the whole point of using [tut](https://github.com/tpolecat/tut)